content/nextflow-on-aws-batch/40_nextflow202/20_simple.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Local Run"
+chapter: false
+weight: 20
+---
+
+
+## Run pipeline 
+
+To check our setup we will run nextflow on the Cloud9 instance, submitting jobs to AWS Batch.
+
+![](/images/nextflow-on-aws-batch/nextflow202/nextflow-test-arch.png)
+
+
+```
+cd ~/environment/nextflow-tutorial
+nextflow run script7.nf -profile batch -w s3://${BUCKET_NAME}
+```
+
+```
+N E X T F L O W  ~  version 20.01.0
+Launching `script7.nf` [awesome_euclid] - revision: ce58523d1d
+R N A S E Q - N F   P I P E L I N E    
+===================================
+transcriptome: /home/ec2-user/environment/nextflow-tutorial/data/ggal/transcriptome.fa
+reads        : /home/ec2-user/environment/nextflow-tutorial/data/ggal/gut_{1,2}.fq
+outdir       : results
```

Comment:
after this line I got "WARN: Unable to create AWS Batch helper class | credentials cannot be null"
and after same time it failed Oops .. something went wrong

Reply:
ok, let's have someone else go through it with the updates - works on my cloud. :)